websites/E/EternalNetwork TM/presence.ts

```diff
@@ -169,6 +169,40 @@ presence.on("UpdateData", async () => {
 
 			break;
 		}
+		case `et-log.${etrnl}`:
+		case `www.et-log.${etrnl}`: {
+			presenceData.smallImageKey = "eternallogistic_logo";
+
+			if (pathname.startsWith("/")) presenceData.state = ttl;
+
+			break;
+		}
+		case `dev.${etrnl}`:
+		case `www.dev.${etrnl}`: {
+			presenceData.smallImageKey = "eternalnetworktm_dev";
+
+			if (pathname.startsWith("/")) {
+				presenceData.state = "Developing new features !";
```

Comment:
```suggestion
				presenceData.state = "Developing new features";
```

Reply:
**Developing** and **Creating** have a different meanings, in programming most of the time, we develop not create ...

Reply:
Your comment is very passive agressive - I know the difference between the two verbs, given I am the maintainer of a large codebase... 

Anyway, creating makes more sense so change it

Reply:
Creating is making more through any means. So agreed with Encrypted.

Reply:
Updated.